curvefs/src/client/fuse_s3_client.cpp

```diff
@@ -194,6 +194,9 @@ CURVEFS_ERROR FuseS3Client::FuseOpWrite(fuse_req_t req, fuse_ino_t ino,
     if (inodeWrapper->GetLengthLocked() < off + *wSize) {
         changeSize = off + *wSize - inodeWrapper->GetLengthLocked();
         inodeWrapper->SetLengthLocked(off + *wSize);
+        if (!FsQuotaChecker::GetInstance().QuotaBytesCheck(changeSize)) {
+            return CURVEFS_ERROR::NO_SPACE;
+        }
```

Comment:
it's better to put this check before `s3Adaptor_->Write`

Reply:
i wanna reuse the existing code to calculate change size here to avoid calling inodeWrapper->GetLengthLocked balabala again